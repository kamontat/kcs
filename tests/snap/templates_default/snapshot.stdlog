00:10:45 [INF] private.base.settings : testing mode has been enabled
00:10:45 [PRT] cmd.tmpl_default : test template is "hello world"
