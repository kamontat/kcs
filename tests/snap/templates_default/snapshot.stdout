00:10:45 [PRT] cmd.tmpl_default : test template is "hello world"
